src/olympia/lib/es/utils.py

```diff
@@ -77,19 +78,26 @@ def create_index(index, config=None):
     Options:
 
     - index: name of the index.
-    - config: if provided, used as the settings option for the
-      ES calls.
+    - config: if provided, used when passing the configuration of the index to
+    ES.
     """
     es = amo_search.get_es()
 
-    if settings.IN_TEST_SUITE:
```

Comment:
This was redundant with `settings_test` and hid the issue we had with the settings.